ntoskrnl/include/internal/inbv.h

```diff
@@ -128,3 +128,34 @@ InbvPortInitialize(
     OUT PULONG PortId,
     IN BOOLEAN IsMMIODevice
 );
+
```

Comment:
So for these values we have to think more about part of a "bootscreen theme", if you see what I mean... Thus I would say they could be placed in a separate header file in the kernel (and in the ntoskrn/inbv directory?)